systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -304,4 +306,75 @@ public void testDeployKafkaOnPersistentStorage() {
+        int expectedKafkaPods = 2;
+        List<String> zkPodStartTime = new ArrayList<>();
+        for (int i = 0; i < expectedZKPods; i++) {
+            zkPodStartTime.add(getResourceCreateTimestamp("pod", zookeeperPodName(clusterName, i)));
+        }
+        List<String> kafkaPodStartTime = new ArrayList<>();
+        for (int i = 0; i < expectedKafkaPods; i++) {
+            kafkaPodStartTime.add(getResourceCreateTimestamp("pod", kafkaPodName(clusterName, i)));
+        }
+        Oc oc = (Oc) this.kubeClient;
+        replaceCm(clusterName, "zookeeper-healthcheck-delay", "23");
+        replaceCm(clusterName, "kafka-healthcheck-delay", "23");
+        replaceCm(clusterName, "KAFKA_DEFAULT_REPLICATION_FACTOR", "1");
+        for (int i = 0; i < expectedZKPods; i++) {
+            kubeClient.waitForResourceUpdate("pod", zookeeperPodName(clusterName, i), zkPodStartTime.get(i));
+            kubeClient.waitForPod(zookeeperPodName(clusterName,  i));
+        }
+        for (int i = 0; i < expectedKafkaPods; i++) {
+            kubeClient.waitForResourceUpdate("pod", kafkaPodName(clusterName, i), kafkaPodStartTime.get(i));
+            kubeClient.waitForPod(kafkaPodName(clusterName,  i));
+        }
+        String configMap = kubeClient.get("cm", clusterName);
+        assertThat(configMap, valueOfCmEquals("zookeeper-healthcheck-delay", "23"));
+        assertThat(configMap, valueOfCmEquals("kafka-healthcheck-delay", "23"));
+        assertThat(configMap, valueOfCmEquals("KAFKA_DEFAULT_REPLICATION_FACTOR", "1"));
+        LOGGER.info("Verified CM and Testing kafka pods");
+        for (int i = 0; i < expectedKafkaPods; i++) {
+            String kafkaPodJson = oc.getResourceAsJson("pod", kafkaPodName(clusterName, i));
+            assertEquals("1", getValueFromJson(kafkaPodJson,
+                    globalVariableJsonPathBuilder("KAFKA_DEFAULT_REPLICATION_FACTOR")));
+            String initialDelaySecondsPath = "$.spec.containers[*].livenessProbe.initialDelaySeconds";
+            assertEquals("23", getValueFromJson(kafkaPodJson, initialDelaySecondsPath));
+        }
+        LOGGER.info("Testing Zookeepers");
+        for (int i = 0; i < expectedZKPods; i++) {
+            String zkPodJson = kubeClient.getResourceAsJson("pod", zookeeperPodName(clusterName, i));
+            String initialDelaySecondsPath = "$.spec.containers[*].livenessProbe.initialDelaySeconds";
+            assertEquals("23", getValueFromJson(zkPodJson, initialDelaySecondsPath));
+        }
+    }
+
+    private String getValueFromJson(String json, String jsonPath) {
+        String value = JsonPath.parse(json).read(jsonPath).toString().replaceAll("\\p{P}", "");
+        return value;
+    }
+
+    private String globalVariableJsonPathBuilder(String variable) {
+        String path = "$.spec.containers[*].env[?(@.name=='" + variable + "')].value";
+        return path;
+    }
+
+    private String getResourceCreateTimestamp(String resourceType, String resourceName) {
```

Comment:
Can this method return type `long` or class `Timestamp`? I think it will be really useful because in this case, we can also check that one value **may be greater than the other**(we can check not only equals or not equals as it is done in the method `waitForResourceUpdate()` in this PR.